src/data/results.html

```diff
@@ -3,34 +3,27 @@
   <head>
     <meta charset='utf-8'>
     <link rel='stylesheet' href='style.css'>
+    <link rel='stylesheet' href='print.css' media='print'>
     <script defer src='template.js'></script>
     <script defer src='utils.js'></script>
     <script defer src='async_storage.js'></script>
     <script defer src='fake-data.js'></script>
-    <script defer src='results-display.js'></script>
+    <script defer src='french-bank-holidays.js'></script>
     <script defer src='form-persistence.js'></script>
+    <script defer src='results-display.js'></script>
     <script defer src='form-interaction.js'></script>
     <script defer src='signature-editor.js'></script>
   </head>
   <body>
-    <section class='choose-form'>
-      <form class='choose-weeks-form hide-future'>
-        <div class='controls'>
-          <button>Generate form</button>
-          <input type='range' class='easy-selector-range' value='0'></input>
-          <label><input type='checkbox' class='show-future-checkbox'> Show future</label>
-        </div>
-      </form>
-    </section>
 
+      <button id='monthDown' onClick="changeMonthDown();">Previous</button>
+      <button id='monthUp' onClick="changeMonthUp();">Next</button>
     <!-- PTO FORM -->
     <section class='pto-form' hidden>
-      <h1>Décompte des heures de travail plafonné en jours sur l'année</h1>
-      <h2 lang='en'>Monitoring form of working hours capped in days over the year</h2>
-      <div class='worker-information'>
-        <div class='who'>Nom (<em lang='en'>name</em>): <span class='last-name' contenteditable='true' data-persist='last-name'>Lastname</span></div>
-        <div>Poste (<em lang='en'>position</em>): <span class='position' contenteditable='true' data-persist='position'>Engineer</span></div>
-      </div>
+      <h1>Feuille auto-déclarative des salariés en forfait-jours</h1>
+      <h2 lang='en'>Auto-declarative form for autonomous executives — 218 days of work per year scheme</h2>
+      <table class='summary'>
+      </table>
```

Comment:
Maybe add an inner comment, like `<!-- Content will be inserted by <summary template function name> -->`